src/components/offers-list/offers-list.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {OfferT} from '../../mocks/offers.ts';
+import PlaceCard from '../place-card/place-card.tsx';
+import {useState} from 'react';
+
+type OffersListProps = {
+  offers: OfferT[];
+}
+
+type Card = {
+  id: string;
+  content: JSX.Element;
+}
+
+function OffersList({offers}: OffersListProps): JSX.Element {
+  const [activeCard, saveIdActiveCard] = useState('Пусто');
```

Comment:
Если начальное значение не определено то undefined или null 
```suggestion
  const [, saveIdActiveCard] = useState('Пусто');
```